### PR TITLE
Provide a default query on an empty query sheet

### PR DESCRIPTION
**Pull request by ricardopinedathen**
This adds a default query when opening the query sheet. I've found myself writing this query time and time again just to see some data on a collection. Saving a few keystrokes today saves fingers for the keystrokes of tomorrow...🖐🙂🖐

### CODE REVIEW

File: src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs
Comment (thomaslevesque) [inline src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs:25]: Could you please put this constant above the other properties and fields, separated by a new line?

File: src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs
Comment (thomaslevesque) [inline src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs:51]: Maybe use `string.IsNullOrEmpty` here? In case `querySheet.Text` is null...

**Review by thomaslevesque — CHANGES_REQUESTED** [review]
Thanks @ricardopinedathen! That's a useful addition indeed.
Could you please make the 2 minor changes I requested before I merge? Thanks!

**Comment by thomaslevesque** [comment]
Actually, I just thought of something: it'd be better to do this here: https://github.com/thomaslevesque/CosmosDBStudio/blob/194e80258fc119de4f72b5fcbd19bf8e3d069903/src/CosmosDBStudio.Model/QuerySheet.cs#L9

Otherwise, loading an existing empty query file will add the default text (not a major problem, but not ideal)

File: src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs
Comment (ricardopinedathen) [reply src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs]: Implemented 👍

File: src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs
Comment (ricardopinedathen) [reply src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs:25]: Implemented 👍

**Comment by ricardopinedathen** [comment]
All changes implemented! Please review

File: src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs
Comment (thomaslevesque) [reply src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs:24]: Thanks. Could you please remove the constant, since it's been moved to QuerySheet? (actually, just undo all the changes to this file)

**Comment by thomaslevesque** [comment]
> All changes implemented! I tested opening and saving query sheets and they seem to be working fine. Opening an empty query sheet opens empty as expected. Please review.

Thanks @ricardopinedathen!
If you can just cleanup the changes in QuerySheetViewModel (see my previous comment), I'll merge right away.

**Comment by ricardopinedathen** [comment]
> > All changes implemented! I tested opening and saving query sheets and they seem to be working fine. Opening an empty query sheet opens empty as expected. Please review.
> 
> Thanks @ricardopinedathen! If you can just cleanup the changes in QuerySheetViewModel (see my previous comment), I'll merge right away.

Done! Thanks!

**Review by thomaslevesque — APPROVED** [review]
Great, thanks again @ricardopinedathen!

**Pull request merged**
